### PR TITLE
refactor: signed credential schema

### PR DESCRIPTION
**Pull request by PatStLouis**
Allow additional optional values

### CODE REVIEW

File: schemas/signed_credential.py [hunk 1]
```diff
@@ -11,10 +11,17 @@ class SignedCredential(Signed, BaseModel):
...
     name: str | dict | None = None
     description: str | dict | None = None
     credentialSubject: dict
+    credentialSchema: List[dict] | dict = None
+    credentialStatus: List[dict] | dict = None
+    refreshService: List[dict] | dict = None
+    renderMethod: List[dict] | dict = None
+    termsOfUse: List[dict] | dict = None
+    relatedResources: List[dict] | dict = None
+    proof: List[dict] | dict = None
```
[the remark this answers is not in the document]
Comment (PatStLouis) [reply schemas/signed_credential.py]: I removed the proof property proof

[the remark this answers is not in the document]
Comment (amanji) [reply schemas/signed_credential.py]: Sorry I meant to only highlight the proof property

File: schemas/signed_credential.py [hunk 2]
```diff
@@ -11,10 +11,17 @@ class SignedCredential(Signed, BaseModel):
         list[str], Body(validation_alias="@context", serialization_alias="@context")
     ]
     id: str
-    type: list[str]
+    type: List[str] | str
     issuer: str | dict
     validFrom: str
-    validUntil: str
+    validUntil: str = None
```
[the remark this answers is not in the document]
Comment (Jsyro) [reply schemas/signed_credential.py:17]: I think you need to allow the None type to make this optional (python 3.10 and up)
https://docs.pydantic.dev/latest/concepts/models/#nested-models

```suggestion
    validUntil: str | None = None
```

[the remark this answers is not in the document]
Comment (amanji) [reply schemas/signed_credential.py:17]: I thought it could be null?

**Pull request merged**